extension/src/popups/MinervaMenu.js

```diff
@@ -1,41 +1,78 @@
-import { renderStudentRankings } from "./MinervaMenuRenderInfo.js";
+import { filterAvailableStoreStickers, renderCurrentStudentInfo, renderStudentRankings } from "./MinervaMenuRenderInfo.js";
 
 function transitionToMainMenu(studentName) {
+    if (!studentName.status) {
+        chrome.tabs.create({url: chrome.extension.getURL("StudentRegistration.html")});
+
+        return;
+    }
     $("#minervaMenuCollapseCard").collapse("show");
```

Comment:
I doubt these two lines do anything good anymore since registration page is now independent.

Queue for deletion next pull request